src/Components/Auth/Login.tsx

```diff
@@ -25,7 +26,11 @@ export const Login = (props: { forgot?: boolean }) => {
     custom_logo_alt,
     custom_description,
   } = useConfig();
-  const dispatch: any = useDispatch();
+  const {
+    data: asset,
+    loading: queryLoading,
+    refetch,
+  } = useQuery(routes.getAsset);
```

Comment:
1. In this login page you don't have to perform any GET requests or need not make any queries on component mount. You'll only need to make POST requests to the login / forget password endpoints.

2. Why exactly are you querying the getAsset API? Was this by mistake?